Adyen/UI/Form/Items/Basic Items/FormAttributedLabelItem.swift

```diff
@@ -0,0 +1,98 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+import UIKit
+
+/// Simple form item that represent a single attributed UILabel element.
+@_spi(AdyenInternal)
+public class FormAttributedLabelItem: FormItem {
+
+    public var subitems: [FormItem] = []
+
+    public init(originalText: String,
+                link: String,
+                style: TextStyle,
+                linkTextStyle: TextStyle,
+                identifier: String? = nil) {
+        self.identifier = identifier
+        self.style = style
+        self.linkTextStyle = linkTextStyle
+        self.originalText = originalText
+        self.link = link
+    }
+
+    public var identifier: String?
+
+    /// The style of the label.
+    public var style: TextStyle
+
+    /// The style of the link text.
+    public var linkTextStyle: TextStyle
+
+    /// The text of the label.
+    public var originalText: String
+
+    // The link label.
+    public var link: String
+
+    public func build(with builder: FormItemViewBuilder) -> AnyFormItemView {
+        let label = ADYLabel()
+        label.numberOfLines = 0
+        label.textAlignment = style.textAlignment
+        label.textColor = style.color
+        label.font = style.font
+        label.backgroundColor = style.backgroundColor
+        let attributedString = NSMutableAttributedString(string: originalText.replacingOccurrences(of: "#", with: " "))
+
+        let linkRanges = linkRangesInString()
+
+        let attributes = createAttributes(from: linkTextStyle)
+        for linkRange in linkRanges {
+            attributedString.addAttributes(attributes, range: linkRange)
+        }
+
+        label.attributedText = attributedString
+        label.accessibilityIdentifier = identifier
+        label.adyen.round(using: style.cornerRounding)
```

Comment:
It is safer to apply `adyen.round(using: )` inside `layoutSubviews()` - corner radius can be sensitive to a view's height/width